src/services/multisig.service.ts

```diff
@@ -0,0 +1,102 @@
+import {Address, AggregateTransaction} from 'nem2-sdk';
```

Comment:
``multisig.service.ts`` looks like a service not specific for the multisig account calls (correct me if I'm wrong). You could opt to rename the service as ``sequentialFetcher.service.ts``.